new-site/src/docs/elements/components/buttons/code.mdx

```diff
@@ -228,9 +222,23 @@ import Playground from '../../../../components/Playground';
 
 </Playground>
 
-
-<PropertyTable />
-
-Note! You can find the full list of properties in the <Link size="M" href="https://hds.hel.fi/storybook/react/?path=/docs/components-button--primary" external>React Storybook.</Link>
+### Packages
+| Package           | Included                      | Storybook link                                                                                                                      | Source link   |
+| ----------------- | ----------------------------- | ----------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| **HDS React**     | <IconCheckCircleFill /> Yes   | <Link size="M" external href="/storybook/react/?path=/story/components-button--primary">View in Storybook</Link>  | <Link size="M" external href="https://github.com/City-of-Helsinki/helsinki-design-system/tree/master/packages/react/src/components/button">View source</Link> |
```

Comment:
Import for IconCheckCircleFill is missing. I can add it to global space.

- [ ] Add IconCheckCircleFill as global so we do not need to import it in every file.